### PR TITLE
Fabiangosebrink/adding schematics

### PR DESCRIPTION
**Pull request by FabianGosebrink**

### CODE REVIEW

File: projects/schematics/src/ng-add/files/auth-config.module.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { AuthModule, OidcConfigService } from 'angular-auth-oidc-client';
+
+export function configureAuth(oidcConfigService: OidcConfigService): () => Promise<any> {
+    return () =>
+        oidcConfigService.withConfig({
+            stsServer: 'please-enter-sts',
+            redirectUrl: window.location.origin,
+            postLogoutRedirectUri: window.location.origin,
+            clientId: 'please-enter-clientId',
+            scope: 'please-enter-scopes',
```
Comment (damienbod) [inline projects/schematics/src/ng-add/files/auth-config.module.ts:11]: Would a comment help here for the scope configuration?

// 'openid profile '  + your scopes, offline_access if you refresh

File: projects/schematics/src/ng-add/actions/add-dependencies.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { addPackageJsonDependency, NodeDependency, NodeDependencyType } from '@schematics/angular/utility/dependencies';
+
+const dependenciesToAdd = [
+    {
+        name: 'angular-auth-oidc-client',
+        version: '11.3.0',
```
Comment (damienbod) [inline projects/schematics/src/ng-add/actions/add-dependencies.ts:7]: Do we have to update this with every version update? Just a note for me

File: projects/schematics/src/collection.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "../../../node_modules/@angular-devkit/schematics/collection-schema.json",
+    "schematics": {
+        "ng-add": {
+            "description": "Add my library to the project.",
```
Comment (damienbod) [inline projects/schematics/src/collection.json:5]: Shouls the description be 

Add OIDC lib to project?

File: projects/schematics/src/ng-add/actions/add-dependencies.ts (hunk 2 above)
Comment (FabianGosebrink) [reply projects/schematics/src/ng-add/actions/add-dependencies.ts:7]: We only have to update this if the mechanism for bootstrapping the auth in our lib changes. but to be consistent I would also update it with every release. I plan to improve that in the next version where the current version is being queries from npm directly via http request.

File: projects/schematics/src/collection.json (hunk 3 above)
Comment (FabianGosebrink) [reply projects/schematics/src/collection.json:5]: Absolutely.

**Pull request merged**